website/members/locale/nl/LC_MESSAGES/django.po

```diff
@@ -680,6 +711,95 @@ msgstr ""
 "verlengd hebt, maar dat dit nog niet is verwerkt: in dat geval hoef je niets "
 "te doen."
 
+#: templates/members/email/information_check.html
+msgid "membership information check"
+msgstr "controle gegevens lidmaatschap"
+
+#: templates/members/email/information_check.html
+msgid ""
+"\n"
+"                    We currently have the following information about you in "
+"our database.<br>\n"
+"                    Via this email we'd like to ask you to check the "
+"accuracy of this data.<br>\n"
+"                    "
+msgstr ""
+"\n"
+"                    Hieronder vind je de (meeste) gegevens die we op dit "
+"moment van je geregistreerd hebben.<br>\n"
+"                    Wil je deze controleren op juistheid?<br>\n"
+"                    "
+
+#: templates/members/email/information_check.html
+msgid "username:"
+msgstr "gebruikersnaam:"
+
+#: templates/members/email/information_check.html
+msgid "name:"
+msgstr "naam:"
+
+#: templates/members/email/information_check.html
+msgid "address:"
+msgstr "adres:"
+
+#: templates/members/email/information_check.html
+msgid "phone:"
+msgstr "telefoon:"
+
+#: templates/members/email/information_check.html
+msgid "date of birth:"
+msgstr "geboortedatum:"
+
+#: templates/members/email/information_check.html
+msgid "email address:"
+msgstr "e-mailadres:"
+
+#: templates/members/email/information_check.html
+msgid "student number:"
+msgstr "studentnummer:"
+
+#: templates/members/email/information_check.html
+msgid "cohort:"
+msgstr "cohort:"
+
+#: templates/members/email/information_check.html
+msgid "programme:"
+msgstr "studie:"
+
+#: templates/members/email/information_check.html
+msgid ""
+"\n"
+"                        <br>\n"
+"                        You can change this data by logging into the website "
+"and using the 'edit profile' page.<br>\n"
+"                        Unable to edit the incorrect information? Then send "
+"an email: info@thalia.nu.<br>\n"
+"                        <br>\n"
+"                        With kind regards,<br>\n"
+"                        <br>\n"
+"                        The board of Study Association Thalia<br>\n"
+"                        <br>\n"
+"                        ————<br>\n"
+"                        <br>\n"
+"                        This email was automatically generated.<br>\n"
+"                    "
+msgstr ""
+"\n"
+"                        <br>\n"
+"                        Je kunt als je ingelogd bent via ‘Bewerk profiel’ de "
+"meeste van deze gegevens aanpassen.<br>\n"
```

Comment:
kan je niet alles aanpassen dat wordt gestuurd?

Reply:
Nope: username, name, birthday are read-only

And even if we could, it would be out-of-scope for this PR. I'm only touching the translations here to make sure the tests pass.